src/app/custom/translations.tsv

```diff
@@ -1102,3 +1102,5 @@
 "routine_arg"	"Field N°"	"Champ N°"
 "routine_arg_add"	"Add a new field"	"Ajouter un nouveau champ"
 "routine_arg_delete"	"Delete this field"	"Supprimer ce champ"
+"label_format_fullscreen"	"Full screen"	"plein écran"
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
"label_format_fullscreen"	"Full screen"	"Plein écran"
```